TrackletAlgorithm/ProjectionCalculator.h

```diff
@@ -250,6 +249,8 @@ template<
     nproj_disk[i] = 0;
   }
 
+  ipage = ipage & 3;
```

Comment:
@aryd Can the magic 3 be defined in terms of `kNBits_MemTPage`?:
```suggestion
  ipage = ipage & ((1 << kNBits_MemTPage) - 1);
```